mozilla-release/browser/base/content/browser.js

```diff
@@ -7209,8 +7209,38 @@ var gIdentityHandler = {
     if (event.target == this._identityPopup) {
       window.addEventListener("focus", this, true);
     }
+
+
+    AddonManager.getAddonByID("https-everywhere@cliqz.com", function(addon){
+      if(addon && addon.isActive){
+        document.getElementById("httsEverywhereGroup").hidden = false;
+        gIdentityHandler.updateHttpsEverywhereUIstate();
+      }
+    });
   },
 
+  updateHttpsEverywhereUIstate: function(){
+    var HTTPS_EVERYWHERE_PREF = "extensions.https_everywhere.globalEnabled";
+    if(Services.prefs.getBoolPref(HTTPS_EVERYWHERE_PREF)){
+      document.getElementById("httsEverywhere-enable").hidden = true;
+      document.getElementById("httsEverywhere-disable").hidden = false;
+    } else {
+      document.getElementById("httsEverywhere-enable").hidden = false;
+      document.getElementById("httsEverywhere-disable").hidden = true;
```

Comment:
Seems like a single button would be more appropriate.
You only need to switch its label dynamically here, taking localized value by means of a `StringBundle`.
There are several of those loaded into browser window. `browser.properties` looks like a good place.